snap7/common.py

```diff
@@ -1,14 +1,16 @@
 import logging
+import os
 import platform
 from ctypes import c_char
 from ctypes.util import find_library
+from typing import Optional, Union
 
 from snap7.exceptions import Snap7Exception
 
 if platform.system() == 'Windows':
     from ctypes import windll as cdll  # type: ignore
 else:
-    from ctypes import cdll
+    from ctypes import cdll  # type: ignore
```

Comment:
why does this require ignoring?

Reply:
It throws a warning saying that it can't determine the type of cdll

Reply:
Well now it doesn't show the same typing error, so i will remove it.

Reply:
This error i'm having, that is hy the ignore.
````
(venv) PS C:\Users\ldapino\Documents\lautaro\python\python-snap7>mypy snap7 test
snap7\common.py:14: error: Incompatible import of "cdll" (imported name has type "LibraryLoader[CDLL]", local name has type "LibraryLoader[WinDLL]")
```